### ISSUE TITLE
Textalign not persisted.

### ISSUE THREAD

**Issue by madc** [dimitrov-adrian/directus-extension-editorjs-interface#24]
When aligning a text with the inline tool (i.e. `center`), it seems the information is not persisted.

![image](https://user-images.githubusercontent.com/343392/148539399-e8e1bee1-d87b-40ee-a90e-f9d51f540ec3.png)

After saving the record, it forgot about the alignment again,

![image](https://user-images.githubusercontent.com/343392/148539513-a483b1ab-440b-477d-b703-0a0f41dada95.png)

The API returns this:
```
{
  "id": "DS9xhnKSHr",
  "type": "paragraph",
  "data": {
    "text": "This should be centered."
  }
},
```

I'm using directus-extension-editorjs-interface v 1.1.0.
Let me know, if I can provide any more information to debug this.

**Comment by madc** [comment]
Wow, that was quick.. thanks! I'll test it as soon as the new version is published.

The paragraph alignment tool module looks great. There is also one called `editorjs-alignment-blocktune` by the same author, ut i could not spot the difference between the too.

**Comment by dimitrov-adrian** [comment]
First is for the paragraphs only, second for other. Anyway, I had added it for headers and quotes as well.

It should be available already in v1.2.0

**Comment by madc** [comment]
I just updated and it works perfectly. Thank you for responding so quickly and putting in the work!

```
{
  "id": "DS9xhnKSHr",
  "type": "paragraph",
  "data": {
	  "text": "This should be centered."
  },
  "tunes": {
	  "alignmentTune": {
		  "alignment": "center"
	  }
  }
},
```
